analyzer/quality-analyzer/README.md

```diff
@@ -0,0 +1,46 @@
+<br/>
+<p align="center">
+    <a href="https://github.com/fasten-project/fasten/actions" alt="GitHub Workflow Status">
+        <img src="https://img.shields.io/github/workflow/status/fasten-project/fasten/Java%20CI?logo=GitHub%20Actions&logoColor=white&style=for-the-badge" /></a>
+    <!-- Here should be a link to Maven repo and version should be pulled from there. -->
+    <a href="https://github.com/fasten-project/fasten/" alt="GitHub Workflow Status">
+                <img src="https://img.shields.io/maven-central/v/fasten/vulnerability?label=version&logo=Apache%20Maven&style=for-the-badge" /></a>
+</p>
+<br/>
+
+
+QualityAnalyzer is FASTEN plugin to store/update code quality metadata from Kafka topic. There is a single Kafka message per single callable produced 
+by [Rapid Plugin](https://github.com/fasten-project/quality-analyzer/tree/master/rapidplugin) using [Lizard](https://github.com/terryyin/lizard). 
+Once a metadata message is consumed, the metadata is stored to respective database depending on the forge (mvn, Debian, PyPI).
+
+It can be used both as a standalone tool and as a part of FASTEN server.
+
+## Arguments
+
+- `-h` `--help` Show this help message and exit.
+- `-d` `--database` Comma separated key-value pairs of Database URLs for connection
+- `-f` `--file` Path to JSON file that contains JSON with Lizard tool generated quality metrics
+
+## Usage 
+
+To start as part of FASTEN server:
+
+```
+java -cp /path/to/projects/fasten/docker/server/server-0.0.1-SNAPSHOT-with-dependencies.jar eu.fasten.server.FastenServer -p /home/fasten/projects/fasten/docker/plugins/ -k localhost:9092 -pl "QualityAnalyzer" -kt "QualityAnalyzer=fasten.RapidPlugin.out" -d "java=jdbc:postgresql://postgres@localhost/dbname,python=jdbc:postgresql://postgres@localhost/pythondbname"
```

Comment:
I think you still need to provide a database password in the environmental variable, don't you?

Reply:
Yes, that is correct, will update the documentation.

Reply:
OK, done